src/test/mod.rs

```diff
@@ -138,6 +149,17 @@ impl TestDatabase {
     }
 }
 
+impl Drop for TestDatabase {
+    fn drop(&mut self) {
+        if let Err(e) = self
+            .conn()
+            .execute(&format!("DROP SCHEMA {} CASCADE;", self.schema), &[])
+        {
+            error!("failed to drop test schema {}: {}", self.schema, e);
```

Comment:
This will get lost in the test logs (docs.rs is kind of noisy). I guess a panic would fail the test for no reason though?

Reply:
A panic would cause the test-runner to abort if the current drop was from a panic, leaving any other test schemas still on the database server. I could make it check `std::thread::panicking` and panic if that wouldn't abort.

Reply:
Hmm good point. I think it's fine to leave it as is, I want to avoid making it unnecessarily complicated. If you see `docs_rs_test_schema` in a database you know where it's from anyway.